meson.build

```diff
@@ -4,6 +4,23 @@
 project('DirectX-Headers', 'cpp', version : '1.610.0',
         default_options : ['cpp_std=c++14'])
 
+cpp = meson.get_compiler('cpp')
+compiler_id = cpp.get_id()
+d3d12_lib = cpp.find_library('d3d12')
+#dxcore is not available in Mingw-w64
+if compiler_id == 'msvc'
+    dxcore_lib = cpp.find_library('dxcore')
+else
+    dxcore_lib = cpp.find_library('dxcore', required: false)
+endif

```

Comment:
If the idea is to require it for MSVC and otherwise never use it, maybe using `dependency('', required: false)` is better in the else branch.

A find_library return object is an ExternalDependency of `type_name == 'library'`, and fully interchangeable with dependency() objects. A dependency with an empty string is special-cased to do nothing, and not invoke any compiler tests or run pkg-config or even emit log lines.

So changing this to `dependency('', required: false)` would be slightly faster, and slightly less verbose, while having the same effect of never being found in mingw.

Reply:
Ideally, MinGW would add DXCore, and this would seamlessly start working.

Reply:
I've gone ahead and changed that if to the single line.  I too hope that dxcore does get added or ported.